tests/test_argparcel.py

```diff
@@ -115,14 +118,16 @@ class MooIntEnum:
     z: ThingyInt | None = None
 
 
+def _get_help_text(cls: type[_typeshed.DataclassInstance]) -> str:
+    """Get the output of `--help` if using argparcel with `cls`."""
+    with contextlib.redirect_stdout(io.StringIO()) as f, pytest.raises(SystemExit):
```

Comment:
Conscious change from `contextlib.suppress` to `pytest.raises` because we don't just want to suppress `SystemExit`, we _expect_ it.